### DISCUSSION TITLE
Add default kernel path as a workspace option

### DISCUSSION THREAD

**Discussion by casassg** [microsoft/vscode-jupyter#9706] [New Ideas]
As a team member, I have a script that automatically creates and bootstraps an environment for my team. This includes the creation of a custom kernel. This makes for more reproducible environment since I need to set certain PYTHONPATH variables to correctly load all modules. However for notebooks, I have to ask each team member to switch their default kernel to the generated kernel (that is if even the kernel is loaded by the extension - see #9703 )

Ideally we have something similar than Python extension: `python.defaultInterpreterPath` but `jupyter.defaultKernelPath`

**Comment by IanMatthewHuff** [comment]
@casassg Thanks for the feedback here. This is something that we're discussing on our end already. The best work item to look at (or add your feedback on) might be this item here: https://github.com/microsoft/vscode-jupyter/issues/9615. In particular this bullet point in it is what you are looking for I believe.

> Allow users to select an item from the kernel list as the default (preferred) for notebooks
> How will this work if Jupyter extension nominates a kernel as the preferred? Which takes precedence?
> How can a user manage this? If they select something as a default, how do they re-set this? Do they need to?
